interview-questions.md

```diff
@@ -4,51 +4,66 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What is object destructuring?
 
-  Your answer:
+  Your answer: this is when instead of putting an object together we are taking it appart.
 
-  Researched answer:
+  Researched answer: is a JavaScript expression that makes it possible to unpack values from arrays, or properties from objects, into distinct variables.
 
 
 
 2. What are the three arguments that .map() can accept?
 
-  Your answer:
+  Your answer: since .map is looking at each indeex individualy i would think that it accepts more than three arguments. ok, i am wrong
 
-  Researched answer:
+  Researched answer: value, index, array 
 
 
 
 3. What is the difference between a class and an object?
 
-  Your answer:
+  Your answer: class is a collection of something while the object is the collection of classes and can have method
 
-  Researched answer:
+  Researched answer: i am so wrong. it is the other way aroud
```

Comment:
you were on the right track. a class is a blueprint for objects and an object is a set of key: value pairs